include/class.search.php

```diff
@@ -984,7 +984,13 @@ static function counts($agent, $cached=true, $criteria=array()) {
             }
 
             if ($Q->constraints && !$empty) {
-                $expr = SqlCase::N()->when(new SqlExpr(new Q($Q->constraints)), new SqlField('ticket_id'));
+                $constraints = $Q->constraints;
+                // Add path_constraints to get the correct counts
+                foreach ($Q->path_constraints as $pc) {
+                    if (!empty($pc[0]->constraints))
```

Comment:
May need to add another foreach for the $pc but still looking into that

Reply:
test annotation counts

Reply:
annotations are counting correctly here 👍🏻

Reply:
each path_constraint is separate so no need for another foreach (thank goodness)